common/achievement/types.ts

```diff
@@ -47,3 +67,58 @@ export type ConditionDataAggregations = {
         createBuckets?: string;
```

Comment:
```suggestion
        bucketCreator?: string;
```

Reply:
I think the previous made more sense. The createBuckets variable is pointing to the values after which the buckets will be created. The bucketCreator will then be selected using these values.